src/main/java/systems/composable/dropwizard/cassandra/connect/AddressTranslatorFactory.java

```diff
@@ -0,0 +1,26 @@
+package systems.composable.dropwizard.cassandra.connect;
+
+import com.datastax.driver.core.policies.AddressTranslator;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+
+import io.dropwizard.jackson.Discoverable;
+
+/**
+ * A service provider interface for creating DataStax {@link AddressTranslator}.
+ *
+ * <p>To create your own, just:
+ *
+ * <ol>
+ *   <li>Create a class which implements {@link systems.composable.dropwizard.cassandra.connect.AddressTranslatorFactory}.
+ *   <li>Annotate it with {@code @JsonTypeName} and give it a unique type name.
+ *   <li>Add a {@code META-INF/services/systems.composable.dropwizard.cassandra.connect.AddressTranslatorFactory} file with your implementation's full class
+ *       name to the class path.
+ * </ol>
+ *
+ * @see EC2MultiRegionAddressTranslatorFactory
+ * @see IdentityTranslatorFactory
+ */
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
+public interface AddressTranslatorFactory extends Discoverable {
+  AddressTranslator build();
```

Comment:
Please use the same indentation (4 spaces, not 2) as existing code to maintain uniformity. The same applies to other code in this PR.